shift_detector/checks/simple_check.py

```diff
@@ -34,8 +35,7 @@ def relative_metric_difference(self, column, metric_name):
             return 0
         # TODO: think about comparison if base value is 0
```

Comment:
Have you thought about that? Please fix this TODO

Reply:
yup, deprecated comment